src/tests/Interop/DllImportSearchPaths/DllImportSearchPathsTest.csproj

```diff
@@ -14,16 +14,24 @@
 
   <Target Name="SetUpSubdirectoryNative" AfterTargets="CopyNativeProjectBinaries">
     <ItemGroup>
-      <_FilesToMove Include="$(OutDir)/libNativeLibrary.*" />
-      <_FilesToMove Include="$(OutDir)/NativeLibrary.*" />
+      <AssembliesToCopyNative Include="$(OutDir)/libNativeLibrary.*" />
```

Comment:
Would `NativeLibrariesToCopy` be a better name for this? This does not seem to be copying managed assemblies.